docs/source/guide/qse-3-options.md

```diff
@@ -0,0 +1,49 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+
+# What additional options are available in QSE?
+In addition to the necessary ingredients already discussed in [How do I use QSE?](qse-1-intro.md), there are a few additional options included in the implementation. 
+
+## Caching Pauli Strings to Expectation Values
+
+Specifically, in order to save runtime, the QSE implementation supports the use of a cache that maps pauli strings to their expectation values. This is taken as an additional parameter in the “execute_with_qse” function reproduced below. 
+
+```
+def execute_with_qse(
+    circuit: QPROGRAM,
+    executor: Union[Executor, Callable[[QPROGRAM], QuantumResult]],
+    check_operators: Sequence[PauliString],
+    code_hamiltonian: Observable,
+    observable: Observable,
+    pauli_string_to_expectation_cache: Dict[PauliString, complex] = {},
+) -> float:
```

Comment:
I don't love the idea of repeating the signature of `execute_with_qse` here as it means we've duplicated information. Could we avoid this by adding a link to the API-doc instead? Open to other suggestions too.